sympy/matrices/tests/test_reductions.py

```diff
@@ -279,6 +280,20 @@ def test_rref():
                 0, 1, 1/(sqrt(x) + x + 1)]):
         assert simplify(i - j).is_zero
 
+    a, b, c, d = symbols('a b c d')
+    A = Matrix([[0, 0], [0, 0], [1, 2], [3, 4]])
+    rhs = Matrix([a, b, c, d])
+    assert A.rref(rhs=rhs) == (Matrix([
+    [1, 0],
+    [0, 1],
+    [0, 0],
+    [0, 0]]), Matrix([
+    [   -2*c + d],
+    [3*c/2 - d/2],
+    [          a],
+    [          b]]), (0, 1, 2, 3))
```

Comment:
The indices of the pivots should be (0, 1)

Reply:
I returned rows instead of cols. Fixed.